src/main/java/frc/robot/commands/AutoSetupCommand

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.VisionSubsystem;
+import frc.robot.subsystems.CommandSwerveDrivetrain;
+import frc.robot.subsystems.LEDSubsystem;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class AutoSetupCommand extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final LEDSubsystem m_led;
+  private final VisionSubsystem m_vision;
+  private Rotation2d m_targetRotation;
+  private Translation2d m_targetTranslation;
+  private final CommandSwerveDrivetrain m_drivetrain;
+
+  /**
+   * Creates a new AutoSetupCommand, which runs left led strips to indicate moving in the x direction,
+   * and runs right led strips to indicate moving in the y direction.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public AutoSetupCommand(LEDSubsystem led, VisionSubsystem vision, CommandSwerveDrivetrain drivetrain) {
+    m_led = led;
+    m_vision = vision;
+    m_drivetrain = drivetrain;
+    addRequirements(drivetrain);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    //for two led strips combined
+    AddressableLEDBuffer m_buffer = new AddressableLEDBuffer(120);
```

Comment:
I think all of this should be encapsulated into the `LEDSubsystem` and not in the command.